scripts/docs/snippetadapter.mjs

```diff
@@ -103,6 +103,7 @@ async function buildSnippets( snippets, paths, constants, imports ) {
 		define: Object.fromEntries( Object.entries( constants ).map( ( [ key, value ] ) => [ key, JSON.stringify( value ) ] ) ),
 		outdir: paths.snippetsOutput,
 		entryNames: '[dir]/[name]/snippet',
+		nodePaths: [ upath.resolve( CKEDITOR5_ROOT_PATH, 'node_modules' ) ],
```

Comment:
```suggestion
		nodePaths: [ 
			// Configure the resolver path so building CKEditor 5 snippets works even if `process.cwd()` points to other directories.
			upath.join( CKEDITOR5_ROOT_PATH, 'node_modules' ) 
	],
```

As `CKEDITOR5_ROOT_PATH` is already an absolute path. See: https://github.com/ckeditor/ckeditor5/blob/b8ec8f0c7e024e8e295d397ede726b665e466467/scripts/constants.mjs#L14